src/passes/ConstHoisting.cpp

```diff
@@ -92,6 +92,10 @@ struct ConstHoisting : public WalkerPass<PostWalker<ConstHoisting>> {
         size = getTypeSize(value.type);
         break;
       }
+      case v128: {
+        assert(false && "v128 not implemented yet");
```

Comment:
Instead of crashing, I think it would be better for the pass to ignore such constants, so `return false;` here would work I believe. Can move the assert text into a TODO comment.